docs/examples/zoom.ipynb

```diff
@@ -0,0 +1,285 @@
+    ")\n",
+    "pipeline = sciline.Pipeline(providers, params=params)\n",
+    "pipeline.set_param_table(masks)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "951aec10",
+   "metadata": {},
+   "source": [
+    "If Mantid is available, we can use it to load data files.\n",
+    "**You must configure the `DataFolder` below to point to the directory containing the data files.**\n",
+    "Otherwise, we fall back to load intermediate data files that have been prepared for the concrete example in this notebook.\n",
+    "If you want to use the workflow with different files you must have Mantid installed:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "66237b3e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    from mantid import ConfigService\n",
+    "    import esssans.isis.mantidio\n",
+    "\n",
+    "    cfg = ConfigService.Instance()\n",
+    "    cfg.setLogLevel(3)  # Silence verbose load via Mantid\n",
+    "\n",
+    "    pipeline[sans.isis.DataFolder] = 'zoom_data'\n",
+    "    for provider in sans.isis.mantidio.providers:\n",
+    "        pipeline.insert(provider)\n",
+    "except ImportError:\n",
+    "    import esssans.isis.data\n",
+    "\n",
+    "    for provider in sans.isis.data.providers:\n",
+    "        pipeline.insert(provider)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "703ffc1e",
+   "metadata": {},
+   "source": [
+    "## Reduction\n",
+    "\n",
+    "### The reduction workflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cae69232",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iofq = pipeline.get(IofQ[SampleRun])\n",
+    "iofq.visualize(graph_attr={'rankdir': 'LR'}, compact=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77687728",
+   "metadata": {},
+   "source": [
+    "### Running the workflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "558327ff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = iofq.compute()\n",
+    "da.plot(norm='log', scale={'Q': 'log'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a7526fc",
+   "metadata": {},
+   "source": [
+    "### Inspecting intermediate results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb922379",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "monitors = (\n",
+    "    WavelengthMonitor[SampleRun, Incident],\n",
+    "    WavelengthMonitor[SampleRun, Transmission],\n",
+    ")\n",
+    "parts = (CleanSummedQ[SampleRun, Numerator], CleanSummedQ[SampleRun, Denominator])\n",
+    "iofqs = (IofQ[SampleRun],)\n",
+    "keys = monitors + (MaskedData[SampleRun],) + parts + iofqs\n",
+    "\n",
+    "results = pipeline.compute(keys)\n",
+    "\n",
+    "display(sc.plot({str(key): results[key] for key in monitors}, norm='log'))\n",
+    "\n",
+    "display(\n",
+    "    scn.instrument_view(\n",
```

Comment:
Be aware that having many instrument views in the docs increases size bloat. For a 2d panel maybe we can make a normal plot?

Reply:
People want to plot calibrated positions though. Is there an easy way of making a 2-D plot with actual positions using the X and Y components?

Reply:
I guess it would also be a job for https://github.com/scipp/plopp/issues/264 ?

Reply:
The histogramming solution applies the masks, which is not great. I think we are lucky in this case since `Y` appears to be constant along tubes, so I can fold instead of histogramming.

Reply:
See update.